server/src/main/java/org/hl7/davinci/endpoint/Utils.java

```diff
@@ -38,4 +39,27 @@ public static URL getApplicationBaseUrl(HttpServletRequest request) {
     }
   }
 
+  public static String stripResourceType(String identifier) {
+    int indexOfDivider = identifier.indexOf('/');
+    if (indexOfDivider+1 == identifier.length()) {
+      // remove the trailing '/'
+      return identifier.substring(0, indexOfDivider);
+    } else {
+      return identifier.substring(indexOfDivider+1);
+    }
+  }
+
+  public static boolean idInSelectionsList(String identifier, List<String> selections) {
+    if (selections.isEmpty()) {
```

Comment:
shouldn't this return false if the list is empty? In the case of an empty list, the identifier we're looking for is not present.

Reply:
Nothing being selected is more of a "anything is allowed".